### PR TITLE
Add methods to statistics.py to fix test_strategy_backtest.py

### PR DESCRIPTION
**Pull request by trb5me**
Basically my goal was to get the test_strategy_backtest.py working

- added update() method to statistics so it can build the returns csv file
- changed a few typos in backtest.py
- added an output/ folder to fixtures

### CODE REVIEW

File: statistics/statistics.py [hunk 1]
```diff
@@ -10,6 +10,27 @@
...
+    def __init__(self, portfolio_handler):
+        self.portf_hand = portfolio_handler
+
+    def update(self, event_time):
+        """On every BAR or TICK event that update() is called, unrealised 
+        pnl is written to the equity.csv file in the output folder"""
+        unrealized_pnls_snapshot = []
+        for ticker in self.portf_hand.portfolio.positions:
+            pt = self.portf_hand.portfolio.positions[ticker]
+            unrealized_pnls_snapshot.append(float(pt.unrealised_pnl))
```
Comment (jacohn16) [inline statistics/statistics.py]: Do we want to use decimal instead of float here?
(https://docs.python.org/2/library/decimal.html)

Comment (trb5me) [inline statistics/statistics.py]: I figured it was okay. Can you even make a pandas DataFrame out of decimal objects? At the moment the csv file it's writing to is only being used for rough strategy evaluation, not accounting stuff.

Comment (trb5me) [inline statistics/statistics.py]: I didn't realize there's a more extensive pull request in front of me. This all will probably have to get thrown away. My fault.

**Comment by jacohn16** [comment]
You and me both  :)    thx for trying anyways.

**Pull request closed without merging**